i18n/locales/en.ts

```diff
@@ -1358,22 +1358,30 @@ const translations = {
         nav_title: 'The input class',
         heading: 'Looking at the Input class implementation',
         paragraph_one:
-          'Bitcoin transaction inputs always point to existing, unspent transaction outputs. Therefore, our Input class has a method <span className="text-green"> from_output() </span> which is used to construct an Input by passing in the output description:',
+          'Here we have code for two classes: an <span className="font-bold">Input</span> class and an <span className="font-bold">Outpoint</span> class.',
```

Comment:
Maybe something like this so right at the top, the distinction is clear. Otherwise its so easy to mis-read 

```suggestion
          'Here we have code for two classes: an <span className="font-bold">Input</span> class and an <span className="font-bold">Outpoint</span> (not "output"!) class.',
```

Reply:
Ah good point!!! It's funny how normal these small distinctions become to us after a while :)